### PR TITLE
Add Head-Like Block Rotation Tutorial

### PR DESCRIPTION
**Pull request by QuazChick**
## Specific (Head-Like) Block Rotation

Guide to creating blocks with more specific rotation, mirroring vanilla mob head mechanics.

Includes a "shell" example block to showcase functionality of end result:

showcase

### CODE REVIEW

File: docs/blocks/head-like-rotation.md [hunk 1]
```diff
@@ -0,0 +1,506 @@
...
+
+The following model for a "shell" block can be used as a reference:
+
+![](/assets/images/blocks/head-like-rotation/model_bones.png)
+
+<Spoiler title="Shell Example Model">
+
+<CodeHeader>RP/models/blocks/shell.geo.json</CodeHeader>
+
+```json
```
Comment (SmokeyStack) [inline docs/blocks/head-like-rotation.md:66]: The geometry bones should have a more descriptive names, from my add-on for example, I used head_45 to notate what angle it is facing

File: docs/blocks/head-like-rotation.md [hunk 2]
```diff
@@ -0,0 +1,506 @@
...
+		"event": "wiki:set_placement"
+	}
+},
+"events": {
+	"wiki:set_placement": {
+		"set_block_property": {
+			// Set the face placed on
+			"wiki:face": "q.block_face == 2 ? 'north' : q.block_face == 3 ? 'south' : q.block_face == 4 ? 'west' : q.block_face == 5 ? 'east' : 'up'",
+			// Now use the long Molang expression from before to set the rotation property
+			"wiki:rotation": "q.block_face == 1 ? { t.positive_head_rot = q.head_y_rotation(0) + 360 * (q.head_y_rotation(0) != Math.abs(q.head_y_rotation(0))); t.block_rotation = Math.round(t.positive_head_rot / 22.5); return t.block_rotation != 16 ? t.block_rotation; };"
```
Comment (SmokeyStack) [inline docs/blocks/head-like-rotation.md:299]: Is this the most compact molang or can it be more simplified?

File: docs/blocks/head-like-rotation.md [hunk 3]
```diff
@@ -0,0 +1,506 @@
...
+    - uspel
+    - QuazChick
+---
+
+
+This tutorial guides you through making a block with proper head-like rotation (e.g. creeper heads and signs), providing examples of a "shell" block with this rotation type.
+
+:::warning BEGINNERS BEWARE!
+
+This tutorial assumes good knowledge of [blocks 1.16.100+](/blocks/blocks-16), [advanced Molang](/documentation/advanced-molang) and creating models in applications such as [Blockbench](https://blockbench.net).
```
Comment (SmokeyStack) [inline docs/blocks/head-like-rotation.md:17]: I would change it to `experimental blocks` instead of `blocks 1.16.100+`

**Review by SmokeyStack — CHANGES_REQUESTED** [review]

**Comment by QuazChick** [comment]
- Changed model bone names to `up_0`, `up_22_5`, `up_45`, `up_67_5`
- `wiki:face` property is now integer, compacting Molang expressions
- Altered introduction warning text `blocks 1.16.100+` -> `experimental blocks`

**Pull request merged**